crypto/rand/rand_lib.c

```diff
@@ -573,20 +574,23 @@ static EVP_RAND_CTX *rand_new_drbg(OSSL_LIB_CTX *libctx, EVP_RAND_CTX *parent,
         return NULL;
     }
 
-    /*
-     * Rather than trying to decode the DRBG settings, just pass them through
-     * and rely on the other end to ignore those it doesn't care about.
-     */
-    cipher = dgbl->rng_cipher != NULL ? dgbl->rng_cipher : "AES-256-CTR";
-    *p++ = OSSL_PARAM_construct_utf8_string(OSSL_DRBG_PARAM_CIPHER,
-                                            cipher, 0);
-    if (dgbl->rng_digest != NULL)
+    settables = EVP_RAND_CTX_settable_params(ctx);
```

Comment:
Why do we need to bother with the `settables` thing? It doesn't seem particularly relevant to this change.

Reply:
This isn't necessary, it could be left out but it's better with.
_If_ we wanted this in 3.0, I'd drop the settables checking.

Post 3.0, I'm fine with it being here.